csi.proto

```diff
@@ -152,8 +152,17 @@ message CreateVolumeRequest {
 
   // End user credentials used to authenticate/authorize volume creation
   // request.
+  // This field contains credential data, for example username and
+  // password. Each key must consist of alphanumeric characters, '-',
+  // '_' or '.'. Each value MUST contain a valid string. An SP MAY
+  // choose to accept binary (non-string) data by using a binary-to-text
+  // encoding scheme, like base64. An SP SHALL advertise the
+  // requirements for credentials in documentation. COs SHALL permit
+  // users to pass through the required credentials. This information is
+  // sensitive and MUST be treated as such (not logged, etc.) by the CO.
   // This field is OPTIONAL.
-  Credentials user_credentials = 6;
+  // The total bytes this field must be less than 1 Mebibyte.
```

Comment:
do we really need to limit this to 1MiB? there's wording in the spec now that declares a default limit for fields declared this way. in this case, the field would be limited by default to 4KiB

Reply:
xref #144